auth-server/modules/auth.install/added/addSaslPlugin.xsl

```diff
@@ -14,8 +14,8 @@
                     <properties>
                         <property name="enableTls" value="true"/>
                         <property name="enableNonTls" value="false"/>
-                        <property name="pemKeyPath" value="/opt/eap/standalone/cert/tls.key" />
-                        <property name="pemCertificatePath" value="/opt/eap/standalone/cert/tls.crt" />
+                        <property name="pemKeyPath" value="${{jboss.home.dir}}/standalone/cert/tls.key" />
+                        <property name="pemCertificatePath" value="${{jboss.home.dir}}/standalone/cert/tls.crt" />
```

Comment:
This double brackets look suspicious. Is this what we want?

Reply:
Yes, without them, the values do not get substituted in.